layouts/partials/event.html

```diff
@@ -3,25 +3,52 @@
     <div class="container">
         <div class="row">
             <div class="col-lg-12 text-center">
-                <h2 class="section-heading">{{ with .Site.Params.event.title }}{{ . }}{{ end }}</h2>
-                <h3 class="section-subheading text-muted">{{ with .Site.Params.event.subtitle }}{{ . | markdownify }}{{ end }}</h3>
+                <h2 class="section-heading">It's a PARTY</h2>
```

Comment:
That just seemed to be the pattern of a lot of hugo sites and themes.  Your templates etc define the content vars, and your content gets defined in toml/md depending on what you're doing.